docs/attributes-registry/k8s.md

```diff
@@ -160,6 +162,33 @@ conflict.
 
 ---
 
+`k8s.container.status.reason` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `` | Reason unknown. | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
Does this mean that the metric `k8s.container.status.reason` will show with value 1 and the attribute value for same should be empty? I am bit confused here